UI/Web/src/app/series-detail/_components/metadata-detail-row/metadata-detail-row.component.html

```diff
@@ -17,21 +17,36 @@
     </span>
 
     @if ((libraryType === LibraryType.Book || libraryType === LibraryType.LightNovel) && mangaFormat !== MangaFormat.PDF) {
-      <span class="word-count me-3">{{t('words-count', {num: readingTimeEntity.wordCount | compactNumber})}}</span>
+      <span class="small-text me-3">{{t('words-count', {num: readingTimeEntity.wordCount | compactNumber})}}</span>
     } @else {
-      <span class="word-count me-3">{{t('pages-count', {num: readingTimeEntity.pages | compactNumber})}}</span>
+      <span class="small-text me-3">{{t('pages-count', {num: readingTimeEntity.pages | compactNumber})}}</span>
     }
 
     @if (hasReadingProgress && readingTimeLeft && readingTimeLeft.avgHours !== 0) {
-      <span class="time-left" [ngbTooltip]="t('time-left-alt')">
+      <span class="small-text" [ngbTooltip]="t('time-left-alt')">
         <i class="fa-solid fa-clock" aria-hidden="true"></i>
         {{readingTimeLeft | readTimeLeft }}
       </span>
     } @else {
-      <span class="time-left" [ngbTooltip]="t('time-to-read-alt')">
+      <span class="small-text" [ngbTooltip]="t('time-to-read-alt')">
         <i class="fa-regular fa-clock" aria-hidden="true"></i>
         {{readingTimeEntity | readTime }}
       </span>
     }
+
+    @if (releaseYear) {
+      <span class="small-text ms-3">
+        <i class="fas fa-calendar"></i>
```

Comment:
You need aria-hidden="true" whenever you are using icons that are presentation-only.